python/thunder/rdds/fileio/imagesloader.py

```diff
@@ -120,27 +169,44 @@ def fromTif(self, dataPath, ext='tif', startIdx=None, stopIdx=None, recursive=Fa
             from thunder.utils.common import pil_to_array
             conversionFcn = pil_to_array  # use our modified version of matplotlib's pil_to_array
 
-        def multitifReader(buf):
+        if nplanes is not None and nplanes <= 0:
```

Comment:
Yes? `nplanes=1` would indicate that you've actually got a 2d image, where each separate page in the multipage tif corresponds to a different image of the same 2d plane (I am struggling to avoid saying "time point" here :) ). There's a test, `test_fromMultipleMultiTimepointTifs`, that specifies `nplanes=1` and which behaves as expected. Before you ask, yes, the shape in this case is 2d (x,y), and not 3d (x,y,1). :)

But I'm not sure if the question I'm answering is the one you're actually asking? Let me know if I'm off track here.